.travis.yml

```diff
@@ -1,20 +1,26 @@
 ---
+sudo: false
 language: ruby
 cache: bundler
 rvm:
-- 2.4.1
-- 2.5.3
+- 2.4.6
```

Comment:
> test against latest versions of gems 

... and rubies :trollface:

Reply:
sigh... thanks

Reply:
well, feels like this has been in the works since 2.5.3 was the latest

Reply:
@kbrock I was commenting on the fact that your commit message said "test against latest versions of gems", but you are also updating the versions of ruby you are testing against.

Again, just trolling.